client/task-list/task-item.tsx

```diff
@@ -24,6 +25,29 @@ type TaskItemProps = {
 	time?: string;
 	content?: string;
 	expanded?: boolean;
+	level?: 'l1' | 'l2' | 'l3';
```

Comment:
Having the `l` prefix seems redundant when supplied to a `level` prop. Can we accept just `1`, `2`, or `3` (and perhaps be lenient on strings since these are added by 3PDs)?

Reply:
Updated in https://github.com/woocommerce/woocommerce-admin/pull/6916/commits/fd91ddbcc95707aafc281f79e3c26477fb505cd2 using integers, I am using `parseInt` after we call the filter for the `3PD` stuff, this way we don't have to worry about any other manual checking on our end.